.github/workflows/main.yml

```diff
@@ -0,0 +1,25 @@
+name: Github Actions
+
+on: [push, pull_request]
+
+jobs:
+  host-x86:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        compiler: [gcc, clang]
+    steps:
+      - name: checkout code
+        uses: actions/checkout@v4
+      - name: Install prerequisite
+        run: |
+          sudo apt-get update -q -y
+          sudo apt-get install -q -y file
+          sudo apt-get install -q -y build-essential
+      - name: Build artifact
```

Comment:
This step is not precise. The following commands conclude `make test-all`. You  should add another step to clarify.

Reply:
In this step, the command `make slimcc` should appear.

Reply:
In next step, the command `make test` should be executed.